lib/cocoapods/generator/xcconfig/aggregate_xcconfig.rb

```diff
@@ -68,12 +68,26 @@ def generate
           # that use Swift. Setting this for the embedded target would
           # cause an App Store rejection because frameworks cannot be embedded
           # in embedded targets.
-          if !target.requires_host_target? && pod_targets.any?(&:uses_swift?)
-            config['EMBEDDED_CONTENT_CONTAINS_SWIFT'] = 'YES'
-            config['ALWAYS_EMBED_SWIFT_STANDARD_LIBRARIES'] = 'YES'
+
+	  # Extract swift_version number
+	  @swift_version = target.build_configurations.build_settings['SWIFT_VERSION']
+          @swift_version_check = 3.0
```

Comment:
Sorry I'm not sure I understand. Reading the Swift version from the build settings is different than checking `xcodebuild -version` because you could have multiple versions of `xcodebuild` available on the system, and whichever one is in use depends on the environment (specifically if you've used `xcode-select --switch`), but not necessarily the project.

But, I'm thinking about this more, and this check won't work if the project isn't a Swift project, since it won't have a swift version set. What if we just check for the existence of the `EMBEDDED_CONTENT_CONTAINS_SWIFT` swift setting? This'll allow the user to upgrade just the way we all did by click on the thing in Xcode and approving the new recommended settings. Then, we'll start being the new `ALWAYS_EMBED_SWIFT_STANDARD_LIBRARIES`, and then CocoaPods can switch to that one. If we don't see the `EMBEDDED_CONTENT_CONTAINS_SWIFT` flag, it won't matter in older Xcode versions, since Xcode will just ignore it. Does this make sense?